lib/parallel_tests/tasks.rb

```diff
@@ -147,10 +147,11 @@ def parse_args(args)
       end
       executable = File.join(File.dirname(__FILE__), '..', '..', 'bin', 'parallel_test')
 
-      command = "#{executable} #{type} --type #{test_framework} " \
+      command = "#{executable} --type #{test_framework} " \
         "-n #{count} "                     \
         "--pattern '#{pattern}' "          \
-        "--test-options '#{options}'"
+        "#{options} "                      \
+        "#{type} "
```

Comment:
This change fixes that `--test-options` is no longer supported and allows full options to be passed. Also fixed additional bug that 'Pass files or folders to run' would be shown and no tests executed.